main.tex

```diff
@@ -239,9 +239,11 @@ \subsection{Astropy development model}
 line count includes comments, as these are often as important for
 maintainability as the code itself.  Without comments there are $142197$ lines
 of code.} contributed by $232$ unique contributors over $19270$ \texttt{git}
-commits.
-The commit distribution \inlinecomment{Tom R}{too vague, maybe 'the distribution of number of commits as a function of ...'} follows an approximate power-law (with log-slope of
-$\approx -0.47$) \inlinecomment{AG}{round to -0.5?  This is astronomy, after all.}, meaning most of the commits were and are contributed by a
+commits. We can construct a distribution of the number of commits
```

Comment:
I still find this wording confusing. On re-reading, I think trying to describe this and stating a power-law index is more confusing than just showing the plot! So, my preference would be to (1) add the plot as a new figure, possibly 2-panel with 2nd panel showing number of committers over time, and (2) rephrase this to say something like:
> Figure~X, left, shows the distribution of number of commits per contributor in log-log space. From this, it's clear that most commits were and are contributed by a relatively small fraction of the contributor pool. Figure~X, right, shows the number of contributors as a function of time since the genesis of the \astropypkg core package.

Reply:
+1 for the figures